templates/commands/goldentest/record.go

```diff
@@ -85,18 +99,17 @@ func (c *RecordCommand) Run(ctx context.Context, args []string) (rErr error) {
 	// Create a temporary directory to validate golden tests rendered with no
 	// error. If any test fails, no data should be written to file system
 	// for atomicity purpose.
-	tempDir, err := renderTestCases(ctx, testCases, c.flags.Location)
+	tempDir, err := renderTestCases(ctx, testCases, templateLocation)
 	if err != nil {
 		return fmt.Errorf("failed to render test cases: %w", err)
```

Comment:
We need some way of identifying which test failed.

Here's what it looks like when spec.yaml parsing fails (notice how it doesn't say which file):

```
$ go run cmd/abc/abc.go templates golden-test record
failed to render test cases: failed to render golden tests: error reading template spec file: error parsing YAML file spec.yaml: at line 26 column 7: unknown field name "foo"; valid choices are [as from on_conflict paths skip]
exit status 1
```

Reply:
I changed the error msg here.
At the same time, I added the logging info to indicate which test case or which template location we are using.